Src/AutoFixture.NUnit3.UnitTest/AutoDataAttributeTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class AutoDataAttributeTest
+    {
+        [Test]
+        public void ExtendsNUnitAttribute()
+        {
+            Assert.That(new AutoDataAttribute(), Is.InstanceOf<NUnitAttribute>());
```

Comment:
Why is it important that the class derives from `NUnitAttribute`?

I tried changing `AutoDataAttribute` so that it derives directly from `Attribute`, and commented out this line here, and the build still succeeds.